public/js/Utils.js

```diff
@@ -59,4 +60,15 @@ function sortedPush(array, value) {
     array.splice(_.sortedIndex(array, value), 0, value);
 }
 
+/**
+ * makeSelector: Add '#' before input string. Makes item selectable in jQuery.
```

Comment:
Nicely explained! Ah, I'm so proud. =P

No seriously, your documentation skills have dramatically improved over the past year!